svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -41,29 +42,14 @@ protected function getMainPluginFileName()
         return 'slick-engagement.php';
     }
 
-/**
- * Called by install() to create any database tables if needed.
- * Best Practice:
- * (1) Prefix all table names with $wpdb->prefix
- * (2) make table names lower case only
- * @return void
- */
     protected function installDatabaseTables()
     {
```

Comment:
do we need to keep these empty functions?

Reply:
For now, yes. I don't want to spend the time to test thoroughly what happens if we remove them.